### PR TITLE
Update to Seed Data and Student section.

### PR DESCRIPTION
**Pull request by trudge-64**
Updated Seed Data with French EL competencies. Added Student section

### CODE REVIEW

File: Web/Pages/Student/Student_Details.cshtml [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+<div class="mrgn-tp-lg">
+    <table class="table table-hover table-responsive table-bordered">
+        <thead>
+            <tr>
+                <th class="bg-primary" colspan="2">@Localizer["position"]: SC-DED 02 Deckhand</th>
+            </tr>
+        </thead>
+        <tbody>
+
+            <tr><th colspan="2" style="background-color:#f2f2f2">@Localizer["certificates"]</th></tr>
```
Comment (ccg-oharan) [inline Web/Pages/Student/Student_Details.cshtml:21]: Is this a mockup page? If not, the text should  be moved to a resource file.

File: Web/Pages/Student/Student_List.cshtml [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+            <tr><td colspan="2">Advanced Marine First Aid</td></tr>
+            <tr><td colspan="2">Bridge Watch Certificate </td></tr>
+            <tr>
+                <th style="background-color:#f2f2f2">@Localizer["knowledge"]</th>
+                <th class="text-center" style="background-color:#f2f2f2">@Localizer["level"]</th>
+            </tr>
+            <tr>
+                <td>
+                    <a href="#1" aria-controls="centred-popup" class="wb-lbx" role="button">Organizational Knowledge</a>
+                    <section id="1" class="mfp-hide modal-dialog modal-content overlay-def">
```
Comment (ccg-oharan) [inline Web/Pages/Student/Student_List.cshtml:39]: Text not in resource file

**Review by ccg-oharan — CHANGES_REQUESTED** [review]

**Comment by trudge-64** [comment]
Sorry, Yes these are Mock-up pages. I didn’t realize I created the Student directory outside of my Mockup folder.

File: Web/Pages/Student/Student_List.cshtml (hunk 2 above)
Comment (trudge-64) [reply Web/Pages/Student/Student_List.cshtml:39]: Yes, these are mockup pages. Not realizing it was outside of the Mockup directory

**Review by ccg-oharan — APPROVED** [review]

**Pull request merged**